File: src/Entity/Ticket.php
<?php

namespace App\Entity;

use Doctrine\ORM\Mapping as ORM;

/**
 * Ticket
 *
 * @ORM\Table(name="ticket", indexes={@ORM\Index(name="idCinema", columns={"idCinema"}), @ORM\Index(name="idEvent", columns={"idEvent"}), @ORM\Index(name="idSalle", columns={"idSalle"})})
 * @ORM\Entity
 */
class Ticket
{
    /**
     * @var int
     *
     * @ORM\Column(name="id", type="integer", nullable=false)
     * @ORM\Id
     * @ORM\GeneratedValue(strategy="IDENTITY")
     */
    private $id;

    /**
     * @var string
     *
     * @ORM\Column(name="idUser", type="string", length=11, nullable=false)
     */
    private $iduser;

    /**
     * @var \DateTime
     *
     * @ORM\Column(name="date", type="date", nullable=false)
     */
    private $date;

    /**
     * @var string
     *
     * @ORM\Column(name="idCinema", type="string", length=30, nullable=false)
     */
    private $idcinema;

    /**
     * @var int
     *
     * @ORM\Column(name="idSalle", type="integer", nullable=false)
     */
    private $idsalle;

    /**
     * @var string
     *
     * @ORM\Column(name="chaise", type="string", length=20, nullable=false)
     */
    private $chaise;

    /**
     * @var string
     *
     * @ORM\Column(name="idEvent", type="string", length=30, nullable=false)
     */
    private $idevent;

    /**
     * @var int
     *
     * @ORM\Column(name="montant", type="integer", nullable=false)
     */
    private $montant;

    public function getId(): ?int
    {
        return $this->id;
    }

    public function getIduser(): ?string
    {
        return $this->iduser;
    }

    public function setIduser(string $iduser): self
    {
        $this->iduser = $iduser;

        return $this;
    }

    public function getDate(): ?\DateTimeInterface
    {
        return $this->date;
    }

    public function setDate(\DateTimeInterface $date): self
    {
        $this->date = $date;

        return $this;
    }

    public function getIdcinema(): ?string
    {
        return $this->idcinema;
    }

    public function setIdcinema(string $idcinema): self
    {
        $this->idcinema = $idcinema;

        return $this;
    }

    public function getIdsalle(): ?int
    {
        return $this->idsalle;
    }

    public function setIdsalle(int $idsalle): self
    {
        $this->idsalle = $idsalle;

        return $this;
    }

    public function getChaise(): ?string
    {
        return $this->chaise;
    }

    public function setChaise(string $chaise): self
    {
        $this->chaise = $chaise;

        return $this;
    }

    public function getIdevent(): ?string
    {
        return $this->idevent;
    }

    public function setIdevent(string $idevent): self
    {
        $this->idevent = $idevent;

        return $this;
    }

    public function getMontant(): ?int
    {
        return $this->montant;
    }

    public function setMontant(int $montant): self
    {
        $this->montant = $montant;

        return $this;
    }


}

File: Artlife-Mobile/src/com/mycompany/entities/Ticket.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package com.mycompany.entities;


import java.util.Date;

/**
 *
 * @author KARIMOOOO
 */
public class Ticket {
  private int id,idSalle,montant ;
 private Date date;
    private String chaise,idCinema,idEvent,idUser;

    public Ticket() {
        
    }

    public Ticket(int idSalle, int montant, String chaise, String idCinema, String idEvent, String idUser) {
        this.idSalle = idSalle;
        this.montant = montant;
        this.chaise = chaise;
        this.idCinema = idCinema;
        this.idEvent = idEvent;
        this.idUser = idUser;
    }

    
    

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public String getIdUser() {
        return idUser;
    }

    public void setIdUser(String idUser) {
        this.idUser = idUser;
    }

    public String getIdCinema() {
        return idCinema;
    }

    public void setIdCinema(String idCinema) {
        this.idCinema = idCinema;
    }

     

    public int getIdSalle() {
        return idSalle;
    }

    public void setIdSalle(int idSalle) {
        this.idSalle = idSalle;
    }

    public String getIdEvent() {
        return idEvent;
    }

    public void setIdEvent(String idEvent) {
        this.idEvent = idEvent;
    }

    public int getMontant() {
        return montant;
    }

    public void setMontant(int montant) {
        this.montant = montant;
    }

    public Date getDate() {
        return date;
    }

    public void setDate(Date date) {
        this.date = date;
     
    }

    public String getChaise() {
        return chaise;
    }

    public void setChaise(String chaise) {
        this.chaise = chaise;
    }

     

    public Ticket(String idUser,String idCinema,int idSalle,String idEvent,int montant,Date date,String chaise) {
        this.idUser = idUser;
        this.idCinema = idCinema;   
        this.idSalle = idSalle;
        this.idEvent = idEvent;
        this.montant = montant;
        this.date = date;
        this.chaise = chaise;
    }

    public Ticket(int id,String idUser,String idCinema,int idSalle,String idEvent,int montant,Date date,String chaise) {
        this.id = id;
        this.idUser = idUser;
        this.idCinema = idCinema;   
        this.idSalle = idSalle;
        this.idEvent = idEvent;
        this.montant = montant;
        this.date = date;
        this.chaise = chaise;
 
    }

    @Override
    public String toString() {
        return "ticket{" + "id=" + id + ", idUser=" + idUser +", idCinema=" + idCinema +", idSalle=" + idSalle +", idEvent=" + idEvent +", montant=" + montant +", date=" + date + ", chaise=" + chaise + '}';
    }
       
}
